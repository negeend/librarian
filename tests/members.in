MEMBER 100000
MEMBER ahsjdbh
MeMbEr 100000
MEMBER BOOKS 100000
member books abcdef
meMbEr bOoks 100000
MEMBER HISTORY 100000
member history hello
mEmbeR HisTorY 100000
ADD MEMBER Sam Sam
ADD MEMBER Peter Griffin
ADD MEMBER Homer Simpson
ADD MEMBER Joji
member 100000
member 100001
member 100002
member 100003
member 100006
MEMBER BOOKS 100000
member books 100003
member books skjfnc
MeMber bOokS 100045
MEMBER HISTORY 100000
member history 100002
mEmber HisTory ajhbfewmkwn
member history 100023
ADD COLLECTION books.csv
RENT 100000 111111
RENT 100000 111112
RENT 100000 111113
RENT 100001 111114
RENT 100001 111115
RENT 100001 111116
RENT 100001 111117
member books 100000
member history 100000
member books 100001
member history 100001
RELINQUISH ALL 100000
RELINQUISH ALL 100001
MEMBER BOOKS 100000
MEMBER HISTORY 100000
MEMBER BOOKS 100001
MEMBER HISTORY 100001
exit