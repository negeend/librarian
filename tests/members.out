user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: No members in system.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000: Sam Sam

user: 100001: Peter Griffin

user: 100002: Homer Simpson

user: 100003: Joji

user: No such member in system.

user: Member not currently renting.

user: Member not currently renting.

user: No such member in system.

user: No such member in system.

user: No rental history for member.

user: No rental history for member.

user: No such member in system.

user: No such member in system.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: Success.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: No rental history for member.

user: The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)

user: No rental history for member.

user: Success.

user: Success.

user: Member not currently renting.

user: To Kill a Mockingbird (Harper Lee)
The Hobbit (J.R.R. Tolkien)
The Iliad (Homer)

user: Member not currently renting.

user: The Odyssey (Homer)
The Hitchhiker's Guide to the Galaxy (Douglas Adams)
So Long and Thanks for all the Fish (Douglas Adams)
Batman Volume 1 (Bob Kane)

user: Ending Library process.
